lib/scope.js

```diff
@@ -1132,18 +972,16 @@ module.exports = {
     return { question_has_id: found_any_question_id, id: id, id_matches: ids_match };
   },  // Ends scope.examinePageID()
 
-  processVar: async function ( scope, { var_data, id }) {
+  processVar: async function ( scope, { var_data, id, non_table=false }) {
```

Comment:
`non_table=false` is a double-negative, which is hard for me to parse at first glance. You could change to `table=true` if that doesn't break other things.